### ISSUE TITLE
can't install @hydrofoil/shaperone-wc (@rdfjs/types)

### ISSUE THREAD

**Issue by vid** [hypermedia-app/shaperone#173]
It looks like there may be an unpublished package?
```
 npm i @hydrofoil/shaperone-wc
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @rdfine/shacl@0.8.4
npm WARN Found: peer @rdfjs/types@">1" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @rdfjs/types@">1" from the root project
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @rdfjs/types@>1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

```
Thanks!

**Comment by tpluscode** [comment]
This can't be right. Let's start with your setup. What version of node/npm are you using? I just npm installed `shaperone-wc` in an empty repository without issues. `npm@6.14.15`

**Comment by vid** [comment]
I'm two versions ahead: ```
$ npm -v
8.1.0
```

OK, if I downgrade to npm@6.14, it works. That doesn't seem like the best practice, though.

**Comment by tpluscode** [comment]
Aha, so turns out I had the peer dependency range wrong. It has to be `>=1` for it to match v1.x of `@rdfjs/types`. npm now automatically installs peer deps so it became a problem.

It's fixed now with `@rdfins/shacl@0.8.5`. You don't need to do anything on you end, just install again.
